bindings/rust/s2n-tls/src/security.rs

```diff
@@ -108,12 +108,15 @@ pub const DEFAULT_TLS13: Policy = policy!("default_tls13");
 #[cfg(feature = "pq")]
 pub const TESTING_PQ: Policy = policy!("PQ-TLS-1-0-2021-05-26");
 
+pub(crate) const TESTING_TLS12: Policy = policy!("20240501");
+
 #[cfg(feature = "pq")]
 pub const DEFAULT_PQ: Policy = policy!("default_pq");
 
 pub const ALL_POLICIES: &[Policy] = &[
     DEFAULT,
     DEFAULT_TLS13,
+    TESTING_TLS12,
```

Comment:
Rust doesn't get mad at you for putting a non-pub struct in a pub list?

Reply:
Nope. I couldn't find docs but experimenting showed that its essentially ok to `pub` export a `pub(crate)`. It does complain if its "private" i.e. not-pub